src/kayenta/stages/kayentaStage/kayentaStageExecutionDetails.controller.ts

```diff
@@ -52,6 +55,28 @@ class KayentaStageExecutionDetailsController {
     // The kayentaStageTransformer pushes related 'runCanary' and 'wait' stages
     // into the 'kayentaCanary' tasks list.
     this.canaryRuns = this.$scope.stage.tasks.filter((t: IExecutionStage) => t.type === RUN_CANARY);
+    this.canaryRuns.forEach((run: any) => {
+      const { canaryConfigId, canaryPipelineExecutionId } = run.context;
+      const metricsAccountName = get(run, ['context', 'metricsAccountName'], '');
+      const canaryRunPromise = getCanaryRun(canaryConfigId, canaryPipelineExecutionId);
```

Comment:
I wonder if Orca should be responsible for annotating the canary execution with this warning so that you don't have to make a network request for the original canary run (and so that the logic for determining the warning doesn't have to live in the client). Especially if we think we might add other warnings later on.

Reply:
Sure thing, I can do that.

Reply:
I like that approach.

General purpose mechanism for providing guidance to canary consumers such that they can iteratively improve their process (hopefully) without having to ask somebody why the results weren't better.